File: app/src/main/java/com/pkfcooperparry/template/data/model/api/request/LoginRequest.java
package com.pkfcooperparry.template.data.model.api.request;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;


public final class LoginRequest {

    @Expose
    @SerializedName("email")
    private String email;

    @Expose
    @SerializedName("password")
    private String password;

    public LoginRequest(String email, String password) {
        this.email = email;
        this.password = password;
    }


    public LoginRequest(String email) {
        this.email = email;
    }

    public String getEmail() {
        return email;
    }

    public String getPassword() {
        return password;
    }

}

File: app/src/main/java/com/televantou/template/data/model/api/request/LoginRequest.kt
package com.televantou.template.data.model.api.request

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName


class LoginRequest {

    @Expose
    @SerializedName("email")
    var email: String? = null
        private set

    @Expose
    @SerializedName("password")
    var password: String? = null

    constructor(email: String, password: String) {
        this.email = email
        this.password = password
    }

    constructor(email: String) {
        this.email = email
    }

}
